pkg/grafana/dashboard-handler.go

```diff
@@ -56,13 +57,23 @@ func (h *DashboardHandler) GetExtension() string {
 	return "json"
 }
 
-func (h *DashboardHandler) newDashboardResource(path, uid, filename string, board Dashboard) grizzly.Resource {
+// APIVersion returns the api version for this resource
+func (h *DashboardHandler) APIVersion() string {
+	return "grafana.com/v1"
+}
```

Comment:
Can we create a spec package for all these definitions? Similar to [tanka/pkg/spec/v1alpha1/environment.go](https://github.com/grafana/tanka/blob/master/pkg/spec/v1alpha1/environment.go)

Reply:
How does this go from golang package to anything that can be consumed outside of golang?